plugins/lsp_d.lua

```diff
@@ -0,0 +1,11 @@
+-- mod-version:3 -- lite-xl 2.1
```

Comment:
You can remove the `-- lite-xl 2.1` part. I know it's in some other plugins, but it's not needed.

Reply:
Ok